### PR TITLE
Add Reolink PIR entity docs

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add Reolink PIR entity docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/117507
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -194,6 +195,8 @@ Depending on the supported features of the camera, switch entities are added for
 - Buzzer on event
 - Email on event
 - FTP upload
+- PIR enabled*
+- PIR reduce false alarm*
```
Comment (davet2001) [inline source/_integrations/reolink.markdown:199]: Are there any descriptions to go with the *ed entries?

I'm not sure what the * means.

**Review by davet2001 — DISMISSED** [review]
I had one minor question on the use of '*'.

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown:199]: Its at the top of the documentation: https://www.home-assistant.io/integrations/reolink/#asterisk--next-to-entities-listed-in-this-documentation

It means that entity is disabled by default.

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Pull request merged**